### PR TITLE
Zcrow donator item

### PR DESCRIPTION
**Pull request by scrcr0**
## About The Pull Request

Adds the custom track pants to the game, locked as a donator item for Zcrow.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl:
add: donator item for Zcrow
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1739,3 +1739,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	icon_state = "anubite_headpiece"
 	worn_icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_y_offset = 4
+
+// Donation Reward for zcrow
+/obj/item/clothing/under/pants/custom_trackpants
+	name = "Tarin's custom sweatpants"
```
Comment (CandleJaxx) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: oh yay i can help with this actually; names for items should be all lowercase, except for when in the loadout dm files for naming structure :)

Comment (scrcr0) [reply modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: Gotcha, will update.

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -1739,3 +1739,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
...
 	worn_y_offset = 4
+
+// Donation Reward for zcrow
+/obj/item/clothing/under/pants/custom_trackpants
+	name = "tarin's custom sweatpants"
+	desc = "A tailored pair of sweatpants (??). Featuring an eye-catching colored stripe and a little more comfort."
+	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/uniform.dmi'
+	icon_state = "zcrow_trackpants"
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform_digi.dmi'
+	body_parts_covered = CHEST|GROIN|LEGS
```
Comment (vinylspiders) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	body_parts_covered = GROIN|LEGS
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Review by vinylspiders — APPROVED** [review]
@GoldenAlpharex

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
You haven't yet met the threshold of $15 in donations required in order to get a donator item. Feel free to contact me on Discord to discuss of any details.

**Comment by scrcr0** [comment]
> You haven't yet met the threshold of $15 in donations required in order to get a donator item. Feel free to contact me on Discord to discuss of any details.

Ah my mistake. I thought I'd hit that at the time of submission. It should be correct as of Dec 1.

**Comment by jjpark-kb** [comment]
I was just about to close this PR-- glad I saw you commented :)

**Comment by jjpark-kb** [comment]
I will be closing the PR-- please contact Golden!

**Pull request closed without merging**